src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -380,5 +394,57 @@ private string GetFramerateStatistics(CompositionManager compositionManager, out
         {
             return GetStatsString("Compositor framerate", compositionManager.FramerateStatistics, out average);
         }
+
+        private bool IsAzureKinectBodyTrackingSDKInstalledInUnity()
+        {
+            if (isAzureKinectBodyTrackingSdkInstalledInUnity == null)
+            {
+                var unityInstallDirectory = AppDomain.CurrentDomain.BaseDirectory;
+                var componentPaths = azureKinectBodyTrackingSdkComponents.Select(component => Path.Combine(unityInstallDirectory, component));
+                isAzureKinectBodyTrackingSdkInstalledInUnity = componentPaths.All(path => File.Exists(path));
+            }
+
+            return isAzureKinectBodyTrackingSdkInstalledInUnity.Value;
+        }
+
+        private void InstallAzureKinectBodyTrackingComponents()
+        {
+            var rootPath = Path.GetDirectoryName(Application.dataPath.Replace(@"/", @"\"));
+            string componentSourceDirectory = null;
+
+            var assets = AssetDatabase.FindAssets(Path.GetFileNameWithoutExtension(azureKinectBodyTrackingSdkComponents[0]));
+            if (assets.Length == 1)
+            {
+                var assetPath = AssetDatabase.GUIDToAssetPath(assets[0]).Replace(@"/", @"\");
+                componentSourceDirectory = Path.GetDirectoryName(Path.Combine(rootPath, assetPath));
+            }
+            else if (assets.Length == 0)
+            {
+                UnityEngine.Debug.LogError($"Failed to find the source components to copy in your Unity installation");
+                return;
+            }
+            else
+            {
+                UnityEngine.Debug.LogError($"Failed to find source directory to install {azureKinectBodyTrackingSdkComponents[0]}: multiple copies were found in the Unity assets directory");
+                return;
+            }
+
+            foreach (var component in azureKinectBodyTrackingSdkComponents)
+            {
+                if (!File.Exists(Path.Combine(componentSourceDirectory, component)))
+                {
+                    UnityEngine.Debug.LogError($"Failed to find component {component}: it is missing from the Unity assets directory {componentSourceDirectory}");
+                    return;
+                }
+            }
+
+            var arguments = $"\"{componentSourceDirectory}\" \"{AppDomain.CurrentDomain.BaseDirectory}\" {string.Join(" ", azureKinectBodyTrackingSdkComponents)}";
+            ProcessStartInfo psi = new ProcessStartInfo("robocopy.exe", arguments);
+            psi.UseShellExecute = true;
+            psi.Verb = "runas";
+            Process.Start(psi).WaitForExit();
+
+            isAzureKinectBodyTrackingSdkInstalledInUnity = null;
+        }
```

Comment:
maybe call IsAzureKinectBodyTrackingSDKInstalledInUnity here and return it's return